### ISSUE TITLE
Any new exercises

### ISSUE THREAD

**Issue by nixfreak** [sergiotapia/nimlings#1]
I think nimlings is great , just wondering if there are any more new exercises?

**Comment by jakeisnt** [comment]
Is there any plan to continue work on this? I'd love to help out if so!

**Comment by sergiotapia** [comment]
Yes sir! I've been on vacation for a bit but more exercises are on the way! Once I finish the exercises I had in mind I would to hear about any other useful advanced exercises. For example, how to use threads properly in a clear realistic example.

**Comment by ynfle** [comment]
Hey guys! I'm a maintainer for the [nim track](https://github.com/exercism/nim) on [Exercism](https://exercism.io) which is a completely open-source community-run platform for learning programming languages. We are currently working on getting the nim track ready for the v3 launch and we'd love your help!

@sergiotapia your stuff is really cool!
